lib/types/subdocument.js

```diff
@@ -161,7 +161,7 @@ Subdocument.prototype.isModified = function(paths, modifiedPaths) {
       paths = this.$__pathRelativeToParent();
     }
 
-    return parent.isModified(paths, modifiedPaths);
+    return parent.$isModified(paths, modifiedPaths);
```

Comment:
@vkarpov15 
Is it possible that `parent` here can be a subdocument? If that's the case it's possible that `$isModified` won't exist on it, we'd need to handle that.

Reply:
It is possible, but subdocuments inherit from documents so they should both have `$isModified()`

Reply:
Then we're safe, thank you.